### PR TITLE
Yesoul S3

### PR DESCRIPTION
**Pull request by eMadman**
this should do it for the S3.  I think the bike mount could be improved with better measurements.

### CODE REVIEW

File: Hardware/Common Assets/Arm/readme.md
Comment (doudar) [inline Hardware/Common Assets/Arm/readme.md:1]: Just double checking - Is the 140 the arm length or the actual measurement? That's Yuge!

**Review by doudar — COMMENTED** [review]
Just check the comment on the arm length.

**Comment by doudar** [comment]
Are these finalized and working?

**Comment by eMadman** [comment]
I just shipped one out recently. I'll check in with the user regarding fit
of the bike mount after it arrives.  I'm pretty sure it's good to go as is
but the fit of the bike mount might not be quite right

**Comment by eMadman** [comment]
Arm length is indeed yuge for the bike

**Comment by doudar** [comment]
> Arm length is indeed yuge for the bike 

Yuge but appropriate, correct?

**Comment by doudar** [comment]
Thanks as usual @eMadman !

**Pull request merged**